### PR TITLE
Adding support for reading ByteArray's from TFRecord files

### PR DESCRIPTION
**Pull request by asketagarwal**
Related Issue: #66 
This is a followup to PR #60 . This can used to read the samples as ByteArrays.(without deserialization)

### CODE REVIEW

**Comment by junshi15** [comment]
Thanks for your contribution. I will take a look tonight.

File: src/main/scala/com/linkedin/spark/datasources/tfrecord/TFRecordDeserializer.scala [hunk 1]
```diff
@@ -14,6 +14,10 @@ import scala.jdk.CollectionConverters._
  */
 class TFRecordDeserializer(dataSchema: StructType) {
 
+  def deserializeByteArray(byteArray: Array[Byte]) = {
```
Comment (catdollar) [inline src/main/scala/com/linkedin/spark/datasources/tfrecord/TFRecordDeserializer.scala]: add return type?

**Review by catdollar — APPROVED** [review]
Looks good to me, left minor comment. Thanks for the contributions!

**Review by junshi15 — APPROVED** [review]
Please address @mizhou-in 's comment.
LGTM.

**Comment by asketagarwal** [comment]
Addressed the comment. Thank you for the quick review !

**Comment by junshi15** [comment]
released as v0.6.1

**Pull request merged**